lib/common.js

```diff
@@ -764,6 +761,36 @@ function readApp(projectId, appId, cb) {
   }
 }
 
+function deleteApp(projectId, appId, cb) {
```

Comment:
👍 Added into the epic a sub-task for we do it for all. 
https://issues.jboss.org/browse/FH-3653